File: frontend/src/app/input/course-section.ts
export class CourseSection {
  static counter: number = 1;
  id: number;
  sectionTitle: string;
  sectionMarkdown: string;

  constructor(sectionTitle: string, sectionMarkdown: string, id: number = CourseSection.counter++) {
    this.id = id;
    this.sectionTitle = sectionTitle;
    this.sectionMarkdown = sectionMarkdown;
  }
}

File: src/main/java/com/zihler/courses/output/CourseSection.java
package com.zihler.courses.output;

public class CourseSection {
    private long id;
    private String sectionTitle;
    private String sectionMarkdown;

    public CourseSection() {

    }

    public CourseSection(long id, String sectionTitle, String sectionMarkdown) {
        this.id = id;
        this.sectionTitle = sectionTitle;
        this.sectionMarkdown = sectionMarkdown;
    }

    public long getId() {
        return id;
    }

    public String getSectionTitle() {
        return sectionTitle;
    }

    public String getSectionMarkdown() {
        return sectionMarkdown;
    }
}
